### ISSUE TITLE
Referral Form Referring Agency 

### ISSUE THREAD

**Issue by Scott123180** [Capping-CPCA/Web-App#169]
- Under Contact information to mirror format of mailing address (_also update format if duplicated across multiple pages_)
![image](https://user-images.githubusercontent.com/9338669/32995239-a900e160-cd3f-11e7-8d19-c175e8612d2d.png)

- The option other should expand a text box allowing for the name of the agency to be input. Also under additional parties involved.
![image](https://user-images.githubusercontent.com/9338669/32995251-ded05802-cd3f-11e7-9285-f1ca6dd151b3.png)

- Input box expansion: not expanding properly on narrower screens.
![image](https://user-images.githubusercontent.com/9338669/32995404-8b01ae90-cd41-11e7-88ce-8dc77588b766.png)

**Comment by Scott123180** [comment]
- Also selecting sex should remove the relationships which do not apply (i.e. select female, so son, father, etc. should not appear). Currently seen under household member, but may be in other places in application as well.
![image](https://user-images.githubusercontent.com/9338669/32995463-b1187aea-cd42-11e7-8956-2b3d7e5c8cc5.png)

**Comment by Scott123180** [comment]
- Checkboxes do not align with referral form:
![image](https://user-images.githubusercontent.com/9338669/32995495-47d7bc0c-cd43-11e7-926b-0b53ce0bc77a.png)

![image](https://user-images.githubusercontent.com/9338669/32995496-4f4376ca-cd43-11e7-8c5c-dcce948260ea.png)

**Comment by sbohner14335** [comment]
All of the formatting issues you can fix by zooming the page out and everything should align correctly. We are aware of the scalability issues, but for now we are going to have to work with it.

As for the "other" drop downs, we had discussed and decided that the selection of "other" not display a field for specification, but we can ask @algozine for a more detailed explanation. As of now our forms are left without a further explanation field.

Finally, I would recommend the relationship drop down feature as a parking lot item for now. It's a good idea / nice to have, but right now we are really just trying to implement edit functionality for all of our forms.

Hope that explained everything!

**Comment by Scott123180** [comment]
@sbohner14335 
For the scalability issues I noticed your form is not using flexbox, something new to bootstrap 4 that helps solve this issue. As of now, the forms are condensed to the right side of their container. I'm going to raise a separate issue for this as it's big task but something that should be done before the final release.

I noticed the other dropdowns while doing the data entry yesterday so we might need to add more agencies to the default list or the add the other field.
